pkgs/by-name/fi/filius/package.nix

```diff
@@ -0,0 +1,115 @@
+{
+  stdenv,
+  lib,
+  fetchzip,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  jre,
+}:
+let
+  # we cache potentially unstable upstream inputs (icon and .zip file) via https://web.archive.org - this is a standard procedure in Nixpkgs
+  icon128 = fetchurl {
+    url = "https://web.archive.org/web/20240623123147/https://dl.flathub.org/repo/appstream/x86_64/icons/128x128/de.lernsoftware_filius.Filius.png";
+    sha256 = "sha256-xES6RWf2KiNsukD6GwTuPyiDI6SseADGLb1AkBSTgd8=";
+  };
+  # Using Gitlab API access to raw data from
+  # https://gitlab.com/filius1/filius/-/tree/master/src/main/resources/gfx?ref_type=heads
+  icon16 = fetchurl {
+    url = "https://web.archive.org/web/20240623134140/https://gitlab.com/api/v4/projects/18855527/repository/files/src%2Fmain%2Fresources%2Fgfx%2Ffilius16.ico/raw?ref=master";
+    sha256 = "sha256-yovHXF4is9n6QR72tIJ7+TRZsQcpcKuXSaCFgMCWZYU=";
+  };
+  icon32 = fetchurl {
+    url = "https://web.archive.org/web/20240623134238/https://gitlab.com/api/v4/projects/18855527/repository/files/src%2Fmain%2Fresources%2Fgfx%2Ffilius32.ico/raw?ref=master";
+    sha256 = "sha256-yovHXF4is9n6QR72tIJ7+TRZsQcpcKuXSaCFgMCWZYU=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "filius";
+
+  # UPDATE instructions
+  #
+  # - Open https://www.lernsoftware-filius.de/Herunterladen and copy download link to clipboard.
+  #   (e.g. https://www.lernsoftware-filius.de/downloads/Setup/filius-2.5.1.zip)
+  # - identify the new version string.
+  version = "2.5.1";
```

Comment:
this is actually not the latest version, [their source code repo](https://gitlab.com/filius1/filius) is at 2.6.0

Reply:
though they didn't release a new zip for that version